popups/scratch-messaging/popup.html

```diff
@@ -67,324 +66,331 @@
   </head>
 
   <body style="display: none" v-show="mounted">
-    <div v-show="!error">
-      <!-- Scratch Team Messages -->
-      <div class="message-type" v-show="stMessages.length" style="border-color: #ff7b26">
-        <div
-          class="message-type-title hover-reverse"
-          @click="messageTypeExtended.stMessages = !messageTypeExtended.stMessages"
-        >
-          <div class="btn-dropdown">
-            <img src="../../images/icons/expand.svg" alt="v" :class="{'reverted': messageTypeExtended.stMessages}" />
-          </div>
-          <span class="message-type-title-text">{{ uiMessages.stMessagesMsg }}</span>
-          <span class="float-right"
-            ><img class="small-icon inverted" src="../../images/icons/notice.svg" /> {{ stMessages.length }}</span
+    <div class="contents">
```

Comment:
Is wrapping everything this div the only change on this file?

Reply:
This changes the position of the scrollbar, but I don't see any problem with that.

Reply:
Yep